kohese-portal/server/boot/routes.ts

```diff
@@ -54,6 +55,15 @@ module.exports = function (app) {
       res.sendFile(path.resolve(clientBundlePath, 'index.html'));
     });
 
+    // set up rate limiter: maximum of five requests per minute
+    var limiter = new RateLimit({
+      windowMs: 1*60*1000, // 1 minute
+      max: 5
```

Comment:
Why just 5/min?  If we are doing a repo sync at startup, you could get 5 requests in the first second.  Need to choose a number that is reasonable, but not too high/low...

Reply:
I left it at the default. I also tested a repo sync at start up and it didn't have any trouble with it.

Express Rate Limit:
"Basic rate-limiting middleware for Express. Use to limit repeated requests to public APIs and/or endpoints such as password reset."

Reply:
Try doing it after a commit has been performed on a different computer. There should be multiple requests required to get back in sync from a computer that was not attached when the commit was performed.  Note the number of requests will increase based on the depth of the change in the tree. So if it is 6 or 7 levels deep, there will be 1-2 requests per level.

Reply:
Imagine if you are not online for a week and there are 100 different trees that are updated.  You would have to wait 20 minutes to get in sync...

Reply:
express-rate-limit Only effects HTML requests not socket requests